### ISSUE TITLE
Bug in importing bulk data

### ISSUE THREAD

**Issue by IanMayo** [debrief/pepys-import#399]
I've wiped the remote Postgres database, and want to shove a load of data in.

Aah, it wasn't completely clear.  I used the Pepys_Admin interface to `import reference data` and `import metadata`.  That may have caused a duplication conflict. 

The import failed when I ran:
 `python -m pepys_import.cli --resolver default --path tests/sample_data/track_files/`

I've done that bulk regularly.  It has just failed while loading a file that we know (from the filename) is _bad_.

Note: if I import just that file, it works.  If I import the hole set of folders, we get the error reported below.

Error log:
```
NMEA Importer working on NMEA_bad.log
 83%|███████████████████████████████████████████████████▋          | 5/6 [00:00<00:00, 11.03it/s]
Traceback (most recent call last):
  File "/usr/local/Cellar/python/3.7.6_1/Frameworks/Python.framework/Versions/3.7/lib/python3.7/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/local/Cellar/python/3.7.6_1/Frameworks/Python.framework/Versions/3.7/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/Users/ian/git/pepys-import/pepys_import/cli.py", line 94, in <module>
    main()
  File "/Users/ian/git/pepys-import/pepys_import/cli.py", line 38, in main
    process(path=args.path, archive=args.archive, db=args.db, resolver=args.resolver)
  File "/Users/ian/git/pepys-import/pepys_import/cli.py", line 90, in process
    processor.process(path, data_store, True)
  File "/Users/ian/git/pepys-import/pepys_import/file/file_processor.py", line 162, in process
    file, current_path, data_store, processed_ctr, import_summary
  File "/Users/ian/git/pepys-import/pepys_import/file/file_processor.py", line 260, in process_file
    data_store, full_path, highlighted_file, datafile, change.change_id
  File "/Users/ian/git/pepys-import/pepys_import/file/importer.py", line 103, in load_this_file
    self._load_this_file(data_store, path, file_object, datafile, change_id)
  File "/Users/ian/git/pepys-import/pepys_import/file/importer.py", line 123, in _load_this_file
    self._load_this_line(data_store, line_number, line, datafile, change_id)
  File "/Users/ian/git/pepys-import/importers/nmea_importer.py", line 107, in _load_this_line
    change_id=change_id,
  File "/Users/ian/git/pepys-import/pepys_import/file/importer.py", line 173, in get_cached_sensor
    change_id=change_id,
  File "/Users/ian/git/pepys-import/pepys_import/core/store/common_db.py", line 217, in get_sensor
    data_store, sensor_name, sensor_type, self.platform_id, privacy, change_id
  File "/Users/ian/git/pepys-import/pepys_import/resolvers/default_resolver.py", line 104, in resolve_sensor
    assert False
AssertionError

```

**Comment by robintw** [comment]
Just a few notes on this error to remind me when I come back to it tomorrow:

- Only occurs when NMEA_Trial is imported first, then NMEA_bad (this can be in the same Pepys Import run or separate runs)
- Only occurs when metadata is imported into database first from pepys admin
- I think a duplicated platform is actually the problem - will investigate tomorrow

Update: Yes, duplicated platform is the issue. Will do temporary fix and explain tomorrow.

**Comment by robintw** [comment]
Fixed by #404.
